manifests/init.pp

```diff
@@ -110,7 +110,6 @@
   Stdlib::Absolutepath $ca_file = $foreman_scap_client::params::ca_file,
   Stdlib::Absolutepath $host_certificate = $foreman_scap_client::params::host_certificate,
   Stdlib::Absolutepath $host_private_key = $foreman_scap_client::params::host_private_key,
-  String $package_name = 'rubygem-foreman_scap_client',
```

Comment:
I thought about leaving this, but found in git history that we dropped support for Debian based package.

Reply:
I'd rather keep it in. We (as theforeman org on github) might have dropped it, but our friends at atix still carry it in their repos https://oss.atix.de/Debian12/pool/ , although the version there is a bit dated.

Reply:
Then I'd need a Puppet master to teach me how one can reassign that variable since Puppet doesn't allow me :/ A different way will be to leave this variable and then add another one as a flag to choose. Or maybe revert https://github.com/theforeman/puppet-foreman_scap_client/commit/c4270a13d23fdc4cb399154688af72536f1b68c8#diff-e78db79abb0b543310767fb9d2814cc4e2408f613de4985186251c70bdd4edbbL6 and add a similar check, but I have no idea how I can propagate `obsolete` variable in there...

@ekohl, since you're the master and the maintainer, can I ask you for a hint maybe?

Reply:
I think I've found a way, but for Debian installation it'll require to provide `obsolete = true` with a different package name to mimik current behavior.

By default it'll try to install the new package with the new config.